cmd/chunks-inspect/loki.go

```diff
@@ -195,11 +244,11 @@ func readUvarint(prevErr error, buf []byte) (uint64, []byte, error) {
 }
 
 func getCompression(format byte, code byte) (Encoding, error) {
-	if format == 1 {
+	if format == chunkFormatV1 {
 		return encGZIP, nil
 	}
 
-	if format == 2 {
+	if format >= chunkFormatV2 {
```

Comment:
changed from `==` to `>=`. is that right?

Reply:
yes it includes V2 and the new v3 so >= works here